pages/about.html

```diff
@@ -0,0 +1,188 @@
+                "Not personal info outlet" and The shield is protecting the cage from the outside world.
+            </p>
+
+            <p class="desktop-extra">
+                The logo competition, a celebration of creativity and innovation, concluded on May 20th this year. We
+                were thrilled
+                to witness the exceptional talent and ingenuity displayed by aspiring graphic designers. Amongst the
+                outstanding
+                entries, Peter Kibet's submission stood out, captivating hearts with its remarkable concept and
+                exquisite execution.
+                Peter's artistic prowess serves as an inspiration to young graphic designers, reminding them of the
+                limitless
+                possibilities that lie within their imagination. Embrace your unique vision, unleash your creativity,
+                and let your
+                designs paint a vibrant world of inspiration and storytelling. The future of graphic design rests in
+                your hands!
+            </p>
+
+
+            <div class="company-img">
+                <img src="./pageAssets/DefenderCage Logo.png" alt="">
+            </div>
+
+            <div class="previous-summits">
+                <h3>See the previous OCD Summits</h3>
+                <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+                <ul>
+                    <li>
+                        <a href="https://summit.creativecommons.org/2018/">
+                            <span>2021 Summit</span>
+                            <p>OCD Summit 2018 in Moscow, Russia.</p>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://summit.creativecommons.org/2019/">
+                            <span>2022 Summit</span>
+                            <p>OCD Summit 2019 in Lisbon, Portugal.</p>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <div id="partners">
+                <h3>Partners</h3>
+                <div class="contributors">
+                    <img src="./pageAssets/airbnb 1.png" alt="">
+                    <img src="./pageAssets/daumkakaologo 1.png" alt="">
+                    <img src="./pageAssets/googlekorea 1.png" alt="">
+                    <img src="./pageAssets/mozilla 1.png" alt="">
+                    <img src="./pageAssets/naver 1.png" alt="">
+                </div>
+            </div>
+        </section>
+
+    </main>
+
+    <footer>
+        <div class="footer-section about">
+            <h1 class="logo-text"><span>©Optimus Cage Defender</span></h1>
+            <p>
+                Optimus Cage Defender is a non-profit organization that aims to provide a secure digital
+                environment for everyone.
+            </p>
+    </footer>
```

Comment:
- [ ] Kindly make sure the footer of your website follows the provided design layout it should contain a logo aligned to the left and text aligned to the right

|Your Website|Provided design|
|-|-|
|![](https://user-images.githubusercontent.com/102757126/241028873-034934cc-a662-4dc6-8e6d-b450415edc35.png)|![Screenshot_20230525_225002](https://github.com/Peter-Kibet/conferencePage/assets/25227022/9649b107-137b-457b-9c7e-59070b57624c)|